zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -2271,7 +2272,8 @@ Moving forward, Four Letter Words will be deprecated, please use
 * *ruok* :
     Tests if server is running in a non-error state. The server
     will respond with imok if it is running. Otherwise it will not
-    respond at all.
+    respond at all, unless it responds with:
+    "ruok is not executed because it is not in the whitelist."
     A response of "imok" does not necessarily indicate that the
     server has joined the quorum, just that the server process is active
```

Comment:
How about this? others lgtm

```
When whitelist enables ruok, the server will respond with `imok` if it is running, otherwise it will not respond at all.
When ruok is disabled, it responds with: "ruok is not executed because it is not in the whitelist."

```